diff --git a/node_modules/expo-av/ios/EXAV.podspec b/node_modules/expo-av/ios/EXAV.podspec
index d057e2d..82eb996 100644
--- a/node_modules/expo-av/ios/EXAV.podspec
+++ b/node_modules/expo-av/ios/EXAV.podspec
@@ -24,6 +24,8 @@ Pod::Spec.new do |s|
   s.pod_target_xcconfig = {
     'DEFINES_MODULE' => 'YES',
     'SWIFT_COMPILATION_MODE' => 'wholemodule',
+    'HEADER_SEARCH_PATHS' => '$(inherited) "${PODS_TARGET_SRCROOT}/../../../ios/ExpoModulesCoreLegacy" "${PODS_ROOT}/Headers/Public/React-Core" "${PODS_ROOT}/Headers/Public/React-Core-prebuilt"',
+    'USER_HEADER_SEARCH_PATHS' => '$(inherited) "${PODS_TARGET_SRCROOT}/../../../ios/ExpoModulesCoreLegacy" "${PODS_ROOT}/Headers/Public/React-Core" "${PODS_ROOT}/Headers/Public/React-Core-prebuilt"',
     "CLANG_CXX_LANGUAGE_STANDARD" => "c++20"
   }
 
